app/cdap/components/LogViewer/TopPanel.tsx

```diff
@@ -144,100 +158,97 @@ const TopPanelView: React.FC<ITopPanelProps> = ({
     dataFetcher.getIncludeSystemLogs()
   );
 
-  function getRawLogsBasePath() {
-    const backendUrl = dataFetcher.getRawLogsUrl();
-    const encodedBackendUrl = encodeURIComponent(backendUrl);
-
-    const url = `/downloadLogs?backendPath=${encodedBackendUrl}`;
-    return url;
-  }
-
-  function getRawLogsUrl() {
-    return `${getRawLogsBasePath()}&type=raw`;
-  }
-
-  function getDownloadLogsUrl() {
-    const fileName = dataFetcher.getDownloadFileName();
-    return `${getRawLogsBasePath()}&type=download&filename=${fileName}.log`;
-  }
-
   function handleToggleSystemLogs() {
     const newState = !includeSystemLogs;
     setLocalIncludeSystemLogs(newState);
     setSystemLogs(newState);
   }
 
-  return (
-    <div className={classes.root} data-cy="log-viewer-top-panel" data-testid="log-viewer-top-panel">
-      <If condition={loading}>
+  if (loading) {
+    return (
+      <div
+        className={classes.root}
+        data-cy="log-viewer-top-panel"
+        data-testid="log-viewer-top-panel"
+      >
         <div className={classes.loadingContainer}>
           <LoadingSVG />
         </div>
-      </If>
-      <Button
-        variant="contained"
-        className={classnames(classes.actionButton, { [classes.disabled]: isPolling })}
-        disabled={isPolling}
-        onClick={getLatestLogs}
-        data-testid="scroll-to-latest"
-      >
-        Scroll to Latest Logs
-        <ArrowDownward className={classes.downArrow} />
-      </Button>
-      <Button
-        variant="contained"
-        className={classes.actionButton}
-        onClick={handleToggleSystemLogs}
-        data-testid="view-advanced-logs"
-      >
-        {includeSystemLogs ? 'Hide' : 'View'} Advanced Logs
-      </Button>
-      <div className={classes.btnGroup}>
+      </div>
+    );
+  }
+
+  return (
+    <div className={classes.root} data-cy="log-viewer-top-panel" data-testid="log-viewer-top-panel">
+      <div className={classes.leftContainer}>
+        <RunLogsStatsChips />
```

Comment:
Where and when this component is shown? Can you add a screenshot for this also please?